lib/internal/child_process.js

```diff
@@ -516,6 +516,12 @@ function setupChannel(target, channel) {
     // a message.
     target._send({ cmd: 'NODE_HANDLE_ACK' }, null, true);
 
+    // It can happen the handle is not received because of some error on
+    // ancillary data reception such as MSG_CTRUNC. In this case don't try to
+    // perform any conversion.
+    if (!handle)
```

Comment:
It's not necessarily an error. For example, connections can be closed by the client while they wait in a queue to be sent from the master to the worker. I think the correct thing to do is advocate that people check their received handles in the worker prior to using them (#13196).